include/alp/reference/matrix.hpp

```diff
@@ -349,12 +349,15 @@ namespace alp {
 	} // namespace internal
 
 	/**
-	 * @brief A reference Matrix is an ALP object.
+	 * Identifies a reference internal matrix is an ALP container.
+	 * \internal \todo Should this relation remain?
 	 */
 	template< typename T >
-	struct is_container< internal::Matrix< T, reference > > {
-		static const constexpr bool value = true;
-	};
+	struct is_container< internal::Matrix< T, reference > > : std::true_type {};
```

Comment:
Should we have two levels of visibility here? internal::is_container usable only internally to differentiate between internal and user-level containers (i.e., Matrix, Vector, Scalar).